wideq/client.py

```diff
@@ -3,14 +3,18 @@
 """
 import json
 import enum
+import logging
 import requests
 import base64
 from collections import namedtuple
+from typing import Any
```

Comment:
If we're going to start depending on the `typing` module, we'll need to ensure we require Python >=3.5. (Might not be worth it to use `Any` where `object` might also work…)

Reply:
Ah, good call, I'm so used to using type hints now, I didn't consider that the library supports 3.4.  Any reason to support 3.4?  I've only ever used >=3.6

Reply:
Not really; mostly just inertia. 3.4 was the first "usable" version of Python 3 IMO.